ibis/backends/tests/test_aggregation.py

```diff
@@ -56,27 +56,25 @@ def mean_udf(s):
 @pytest.mark.parametrize(
     ('result_fn', 'expected_fn', 'expected_col'), aggregate_test_params,
 )
-@pytest.mark.skip_backends(
-    ['dask']
-)  # TODO - aggregations - #2553 (and pd.concat)
 @pytest.mark.xfail_unsupported
 def test_aggregate(
     backend, alltypes, df, result_fn, expected_fn, expected_col
 ):
     expr = alltypes.aggregate(tmp=result_fn)
     result = expr.execute()
 
+    if backend.name() == 'dask':
```

Comment:
maybe instead skip for dask and add a dask specific test? not sure what the pattern is for others like this

Reply:
or can have the backend assertion do t he compute?

Reply:
Yeah - the problem here specifically was the use of a lambda in `agg` which pandas knew how to handle but dask did not. The bigger problem I guess is using pandas inside the test itself.

Reply:
ok can we instead have a property / function on the backend testing class that that allows us to do this automatically? (e.g. for all others this is a no-op).

Reply:
I've replaced all of the checks except for one with a transparent `pandas_df` fixture. 

I left the one stopping us from attempting to sort result in dask. I'll remove that when I tackle sorting.